features/polyExplorer/src/screens/entityDetails/entityDetails.jsx

```diff
@@ -15,9 +15,15 @@ const EntityDetails = () => {
     const { selectedEntityObject, entityObjectByPpid } =
         useContext(ExplorerContext);
     const entity = selectedEntityObject;
+    const years = [2015, 2016, 2017, 2018, 2019];
```

Comment:
Why these years?

Reply:
I don't know, it's the years that were decided to be shown by the inventors of these charts I guess

Reply:
OK, but how does this solve the problem? Before, they weren't filtered by year, right?

Reply:
In the chart itself, the years are also being filtered to show either a single bar or the "no-data" placeholder.
But in this parent-element I added the year-filter to have a parameter for wether  the whole revenue-section should be shown or not